cli/server/previewServer.js

```diff
@@ -30,6 +30,26 @@ const PUBLIC_CONFIG_BASENAME = 'cache-public.config';
 const PUBLIC_CONFIG_PATH = path.join(CACHE_DIRECTORY, PUBLIC_CONFIG_BASENAME);
 const FAV_ICON_PATH = './node_modules/jest-preview/cli/server/favicon.ico';
 
+const updateNotifier = require('update-notifier');
+const chalk = require('chalk'); // Already installed in `update-notifier`
+
+// Checks for available update
+const notifier = updateNotifier({
+  pkg: require('../../package.json'),
+  updateCheckInterval: 0, // How often to check for updates
```

Comment:
What `0` mean, can you elaboragte? Thanks

Reply:
@nvh95 That is an interval threshold (in millisecond). Whenever users initiate the notifier and it's not within the interval threshold, it will asyncly check with npm in the background, then persist the result. The next time (within the interval threshold), the result will be loaded without sending any request to npm. Set `updateCheckInterval` to `0` means the notifier will check with npm every time. IMO, It doesn't affect startup performance at all so I choose `0`, or 1 day, still fine.

Reply:
Thanks.